src/libs/actions/connections/QuickbooksOnline.ts

```diff
@@ -21,7 +22,11 @@ function getQuickbooksOnlineSetupLink(policyID: string) {
     return commandURL + new URLSearchParams(params).toString();
 }
 
-function updateQuickbooksOnlineEnableNewCategories(policyID: string, settingValue: boolean) {
+function updateManyQuickbooksOnyxData<TConfigUpdate extends Partial<Connections['quickbooksOnline']['config']>>(
```

Comment:
```suggestion
function buildOnyxDataForMultipleQuickbooksConfigurations<TConfigUpdate extends Partial<Connections['quickbooksOnline']['config']>>(
```

NAB! This is also not a good name but is a little clearer?